src/Xamarin.Android.Build.Tasks/Tasks/LinkApplicationSharedLibraries.cs

```diff
@@ -97,9 +103,22 @@ void RunLinker (Config config)
 
 				if (proc.ExitCode != 0) {
 					LogCodedError ("XA3007", Properties.Resources.XA3007, Path.GetFileName (config.OutputSharedLibrary));
+					LogErrors ("stdout", stdoutLines);
```

Comment:
Not sure what the best thing to do here is. These errors will not be "attached" to the XA3007. they will be reported as different errors per line. 

Should we instead make the `Properties.Resources.XA3007` take an additional argument which is the output of the tool?

Reply:
It, somewhat, mimics [this behavior](https://github.com/xamarin/xamarin-android/commit/b002dc38196a0bedb18c82d228056e1ba108459c). If we added the new argument, as you suggest, would it retain the newlines from the outputs?

Reply:
@dellis1972 my hope was that the errors will be presented together, so that even though the coded error will be "separate", the following lines would be shown just below it, in the correct context.

Reply:
@jonpryor do we have an old comment/commit message that talks about multiline errors and how they show up in Visual Studio? I think we might be able to just append all these messages in a single error message?

Reply:
> @jonpryor do we have an old comment/commit message that talks about multiline errors and how they show up in Visual Studio? I think we might be able to just append all these messages in a single error message?

If the appended content preserves newlines, then sure, I'll be fine with it. Otherwise I'd rather do what this PR currently does.

Reply:
If we do log them seperately it would be better if it had the same Error code (XA3007)  So maybe use `Log.LogCodedError` instead.

Reply:
@jonathanpeppers [asked](https://github.com/xamarin/xamarin-android/pull/7566#discussion_r1026538064):

> do we have an old comment/commit message that talks about multiline errors and how they show up in Visual Studio?

We have this:

https://github.com/xamarin/xamarin-android/pull/2905#issuecomment-479599104

**CreateErrorsItemGroupSingleLine** was the best looking approach, which joins all lines together with `\n` between them.  This appears to be the approach currently taken in this PR.